imap_data_access/cli.py

```diff
@@ -380,7 +380,7 @@ def add_query_args(subparser: ArgumentParser) -> None:
         required=False,
         help="Version of the product in the format 'v001'."
         " Must have one other parameter to run."
-        " Passing 'latest' will return latest version of a file",
+        " Passing 'latest' will return latest version of a file per day",
```

Comment:
```suggestion
        " Passing 'latest' will return latest version of a file per start_date",
```

I think this is probably intuitive enough but you might want to explicitly use "start_date" here to avoid any confusion of what is meant by "day".